user/pages/03.commerce2/02.developer-guide/02.install-update/02.installation/docs.md

```diff
@@ -41,13 +23,10 @@ Requirements
 
  Tips:
 
-   The `bin` folder contains [Drupal Console] binary.
-   The `web` folder represents the document root.
-   Composer commands are always run from the site root (`mystore` in this case).
-   Downloading additional modules:   `composer require "drupal/devel:1.x-dev"`
-   Updating an existing module: `composer update drupal/address -–with-dependencies`
-
- See the `project-base README`_ for more details.
+ - The `bin` folder contains [Drupal Console] binary.
+ - The `web` folder represents the document root.
+ - Composer commands are always run from the site root (`mystore` in this case).
+ - See the `project-base README`_ for more details.
```

Comment:
Should we document here how to convert `web` to `docroot` if on Acquia? Or maybe there's already a resource for the drupal-composer project on that.

Reply:
Ideally we'd link to an external resource, since it's not Commerce specific.

Reply:
Here's an issue https://github.com/drupal-composer/drupal-project/issues/64

The answers are in there

Reply:
I think this specific comment is best: https://github.com/drupal-composer/drupal-project/issues/64#issuecomment-206455356